packages/arb-bridge-eth/contracts/rollup/facets/RollupUser.sol

```diff
@@ -369,15 +368,18 @@ abstract contract AbsRollupUserFacet is RollupBase, IRollupUser {
         uint256 remainingLoserStake = amountStaked(losingStaker);
         uint256 winnerStake = amountStaked(winningStaker);
         if (remainingLoserStake > winnerStake) {
+            // If loser has a higher stake than the winner, refund the difference
             remainingLoserStake = remainingLoserStake.sub(reduceStakeTo(losingStaker, winnerStake));
         }
 
+        // Reward the winner with half the remaining stake
         uint256 amountWon = remainingLoserStake / 2;
         increaseStakeBy(winningStaker, amountWon);
         remainingLoserStake = remainingLoserStake.sub(amountWon);
         clearChallenge(winningStaker);
-
+        // Send the other half to the owner address
```

Comment:
Technically we don't send funds to the owner address. Instead we make them withdraw-able.

Reply:
ye, maybe "credit to the owner address"